bookkeeper-benchmark/src/main/java/org/apache/bookkeeper/benchmark/BenchBookie.java

```diff
@@ -175,7 +176,7 @@ public static void main(String[] args)
             toSend.writeLong(entry);
             toSend.writerIndex(toSend.capacity());
             bc.addEntry(new BookieSocketAddress(addr, port), ledger, new byte[20],
-                        entry, toSend, tc, null, BookieProtocol.FLAG_NONE);
+                        entry, toSend, tc, null, BookieProtocol.FLAG_NONE, LedgerType.FORCE_ON_JOURNAL);
```

Comment:
line 179, 197, 217 need to use the ledger type bit in metadata, no?

Reply:
I think this is a tricky point. The replicator needs to have guarantee that entry is really persisted.
This was a point were I thought that we needed a per entry flag and not a ledger wise flag.
Do you think that replicator should not wait for fsync?

Reply:
Sorry bad line for comment. Here we should use but from metadata. Thanks

Reply:
so what is the decision? Are we using bit from metadata or not?

Reply:
This is BenchBookie, it only creates LedgerType.FORCE_ON_JOURNAL ledgers.
I tried to do a refactor but this will be a big change in this class. I think it is not important to rewrite this class in this patch.
Mabye it would be better to create an issue on benchmarks in order to support LedgerType and force()

Reply:
okay for me